pages.fr/common/bundle.md

```diff
@@ -0,0 +1,36 @@
+# bundle
+
+> Gestionnaire de dépendances pour le langage de programmation Ruby.
+> Plus d'informations : <https://bundler.io/man/bundle.1.html>.
+
+- Installe toutes les gems définies dans le `Gemfile` attendu dans le répertoire de travail :
+
+`bundle install`
+
+- Exécute une commande dans le contexte du bundle actuel :
+
+`bundle exec {{commande}} {{arguments}}`
+
+- Mets à jour toutes les gems selon les règles définies dans le `Gemfile` et régénére le fichier `Gemfile.lock` :
+
+`bundle update`
+
+- Mets à jour une ou plusieurs gem(s) spécifique(s) définie(s) dans le `Gemfile` :
+
+`bundle update {{nom_de_la_gem}} {{nom_de_la_gem}}`
+
+- Mets à jour une ou plusieurs gem(s) spécifique(s) définie(s) dans le `Gemfile` mais seulement vers la prochaine version de patch :
+
+`bundle update --patch {{nom_de_la_gem}} {{nom_de_la_gem}}`
+
+- Mets à jour toutes les gem(s) du groupe donné dans le `Gemfile` :
+
+`bundle update --group {{nom_groupe}}`
+
+- Liste les gem(s) installée(s) dans le `Gemfile` avec les nouvelles versions disponibles :
+
+`bundle outdated`
+
+- Crée un nouveau squelette de gemme :
```

Comment:
for the rest of the file you use `gem`
But for this line you use `gemme`

I was confused and I had a look at the documentation and they use the term `gem` in french 

![image](https://user-images.githubusercontent.com/9024389/193441753-3f65f737-2949-4dcd-a01c-28d64f54e68e.png)

I think it's ok to use `gem` instead of `gemme` 

```suggestion
- Crée un nouveau squelette de gem :
```

Reply:
Yes, `a gem` translate to `un gemme` in French, but here `gem` is not a common name but a proper name. I missed this one sorry.